### PR TITLE
Cd 57 admin global configuration fields backend

### PR DESCRIPTION
**Pull request by BlacyMacwan**

### CODE REVIEW

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs [hunk 1]
```diff
@@ -1,24 +1,89 @@
...
+		[HttpGet]
+		public async Task<IActionResult> GetAllGlobalConfigurationFields()
+		{
+			try
+			{
+				return await globalConfigurationFieldsService.GetAllGlobalConfigurationFields();
+			}
+			catch (Exception ex)
+			{
+				return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs:40]: need to change catch return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs [hunk 2]
```diff
@@ -1,24 +1,89 @@
...
+		{
+			if (ModelState.IsValid)
+			{
+				try
+				{
+					return await globalConfigurationFieldsService.CreateOrUpdateGlobalConfigurationFields(globalConfigurationFields);
+				}
+				catch (Exception ex)
+				{
+					return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs:60]: need to change catch return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs [hunk 3]
```diff
@@ -1,24 +1,89 @@
...
+				}
+			}
+			return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.BadRequest, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.BadRequest.GetDescriptionAttribute(), Data = SystemMessages.msgBadRequest });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs:63]: need to change return response message

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs [hunk 4]
```diff
@@ -1,24 +1,89 @@
...
+		{
+			if (ModelState.IsValid)
+			{
+				try
+				{
+					return await globalConfigurationFieldsService.CreateOrUpdateGlobalConfigurationFields(globalConfigurationFields);
+				}
+				catch (Exception ex)
+				{
+					return new JsonResult(new CustomResponse<Exception>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.InternalServerError, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.InternalServerError.GetDescriptionAttribute(), Data = ex });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs:82]: need to change catch return response

File: CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs [hunk 5]
```diff
@@ -1,24 +1,89 @@
...
+				}
+			}
+			return new JsonResult(new CustomResponse<string>() { StatusCode = (int)HttpStatusCodesAndMessages.HttpStatus.BadRequest, Result = false, Message = HttpStatusCodesAndMessages.HttpStatus.BadRequest.GetDescriptionAttribute(), Data = SystemMessages.msgBadRequest });
```
Comment (GohilAmit) [inline CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs:85]: need to change return response message

Comment (BlacyMacwan) [reply CIR/Controllers/GlobalConfiguration/GlobalConfigurationFieldsController.cs]: All messages response changes completed.

**Pull request closed without merging**